specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabricManagedClusters/preview/2021-01-01-preview/managedapplication.json

```diff
@@ -0,0 +1,2575 @@
+        },
+        "reservationCapacity": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The node reservation capacity for Service Fabric application.\nThis is the amount of load which is reserved on nodes which have instances of this application.\nIf MinimumNodes is specified, then the product of these values will be the capacity reserved in the cluster for the application.\nIf set to zero, no capacity is reserved for this metric.\nWhen setting application capacity or when updating application capacity; this value must be smaller than or equal to MaximumCapacity for each metric.\n"
+        },
+        "totalApplicationCapacity": {
+          "description": "The total metric capacity for Service Fabric application.\nThis is the total metric capacity for this application in the cluster. Service Fabric will try to limit the sum of loads of services within the application to this value.\nWhen creating a new application with application capacity defined, the product of MaximumNodes and MaximumCapacity must always be smaller than or equal to this value.\n",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "ApplicationMetricList": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ApplicationMetric"
+      },
+      "description": "List of application capacity metric description."
+    },
+    "ApplicationParameterList": {
+      "type": "object",
+      "description": "List of application parameters with overridden values from their default values specified in the application manifest.",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "ApplicationResource": {
+      "description": "The application resource.",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/ManagedIdentity"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApplicationResourceProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "ApplicationResourceList": {
+      "description": "The list of application resources.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of application list results if there are any.",
+          "readOnly": true
+        }
+      }
+    },
+    "ApplicationResourceProperties": {
+      "description": "The application resource properties.",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current deployment or provisioning state, which only appears in the response"
+        },
+        "typeName": {
```

Comment:
Is this a reference to an applicationTypes resource?  If so, it should use the full resource ID of the applicationTypes resource (same for typeVersion)

Reply:
thanks for the suggestion we changed it to the full resource ID